nodes/src/nodes/dataset_cobalt/IGlobal.py

```
@@ -0,0 +1,165 @@
+
+        try:
+            items = self._loader.load()
+            debug(f'Cobalt Dataset Global: Loaded {len(items)} raw items')
+        except FileNotFoundError as e:
+            warning(f'Cobalt Dataset Global: {str(e)}')
+            items = []
+        except ValueError as e:
+            warning(f'Cobalt Dataset Global: {str(e)}')
+            items = []
+        except ImportError as e:
+            warning(f'Cobalt Dataset Global: Failed to import cobalt library: {str(e)}')
```

Ruff (0.15.7)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)